Voltaire/Controllers/Messages/SendToGuild.cs

```diff
@@ -56,16 +56,27 @@ public static async Task LookupAndSendAsync(SocketGuild guild, ShardedCommandCon
                 return;
             }
 
-            if(!IncrementAndCheckMessageLimit.Perform(dbGuild, db))
+            if (!IncrementAndCheckMessageLimit.Perform(dbGuild, db))
             {
                 await Send.SendErrorWithDeleteReaction(context, "This server has reached its limit of 50 messages for the month. To lift this limit, ask an admin or moderator to upgrade your server to Voltaire Pro. (This can be done via the `!volt pro` command.)");
                 return;
             }
 
             var prefix = PrefixHelper.ComputePrefix(context, dbGuild);
             var channel = candidateChannels.OrderBy(x => x.Name.Length).First();
-            var messageFunction = Send.SendMessageToChannel(channel, replyable, context, dbGuild.UseEmbed);
-            await messageFunction(prefix, message);
+            if (context.Message.Attachments.Any())
+            {
+                for (int attachmentIndex = 0; attachmentIndex < context.Message.Attachments.Count; attachmentIndex++)
+                {
+                    var messageFunction = Send.SendMessageToChannel(channel, replyable, context, dbGuild.UseEmbed, attachmentIndex);
+                    await messageFunction(prefix, message);
```

Comment:
This was the case until I added the ternary operations in the SendFileAsync lines.
My implementation of the attachmentIndex was a lazy fix to retrieve the index of the current attachment that was being iterated, to ensure that the attachments were being send in order.
Initially, if the `message` was supplied with each request, it would repost the text alongside each attachment, however I used the attachment index `attachmentIndex == 0 ? message : ""` to remove it.